app/app.go

```diff
@@ -171,7 +180,7 @@ var (
 	ModuleBasics = module.NewBasicManager(
 		auth.AppModuleBasic{},
 		genutil.AppModuleBasic{},
-		bank.AppModuleBasic{},
+		sdkbank.AppModuleBasic{},
```

Comment:
Why use native sdk bank instead of alliance bank here?

Reply:
alliance bank module doesn't have own implementation for AppModuleBasic so I used sdk bank's one.